### PR TITLE
WU binding: example updated in the documentation

### PR DESCRIPTION
**Pull request by lolodomo**
It is not allowed to use "Location" as an item name.

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.weatherunderground/README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ demo.items:
 String Conditions "Conditions [%s]" {channel="weatherunderground:weather:CDG:current#conditions"}
 Image Icon "Icon" {channel="weatherunderground:weather:CDG:current#icon"}
 DateTime ObservationTime "Observation time [%1$tH:%1$tM]" <clock>  {channel="weatherunderground:weather:CDG:current#observationTime"}
-String Location "Location [%s]" {channel="weatherunderground:weather:CDG:current#location"}
+String ObservationLocation "Location [%s]" {channel="weatherunderground:weather:CDG:current#location"}
```
Comment (htreu) [inline extensions/binding/org.eclipse.smarthome.binding.weatherunderground/README.md]: I dont want to make too much of a discussion here, but what does `ObservationLocation` actually mean?

Comment (lolodomo) [inline extensions/binding/org.eclipse.smarthome.binding.weatherunderground/README.md]: Location of xeather observation.
It is the terminology of WU: https://www.wunderground.com/weather/api/d/docs?d=data/conditions

Comment (htreu) [inline extensions/binding/org.eclipse.smarthome.binding.weatherunderground/README.md]: thanks for clarification.

**Comment by sjka** [comment]
Thanks!

**Comment by kaikreuzer** [comment]
> It is not allowed to use "Location" as an item name.

@lolodomo Just as a small correction: It is allowed, but when using keywords as names, they have to be escaped as `^Location`.

**Pull request merged**